src/class/object_genome_builds.php

```diff
@@ -115,6 +116,33 @@ function prepareData ($zData = '', $sView = 'list')
 
         if ($sView == 'list') {
             $zData['created_date_'] = substr($zData['created_date'], 0, 10);
+
+        } elseif ($sView == 'entry') {
+            global $_DB;
+
+            $sSlash = ($zData['column_suffix']) ? '/' : '';
+
+            $iPercentComplete = $_DB->query(
+                'SELECT (mapped_variants / all_variants) * 100 FROM
+                    (SELECT COUNT(id) AS mapped_variants FROM lovd_variants
```

Comment:
Note that this query contains a hardcoded table name. Luckily, our table prefixes differ, and I got a query error :sweat_smile: 
I'll fix it when I move the query and merge it with the VE's query.